src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -221,6 +225,9 @@ public Measure createDraft(
     measureDraft.setCreatedAt(now);
     measureDraft.setLastModifiedAt(now);
     measureDraft.setCreatedBy(username);
+    if (!CollectionUtils.isEmpty(measure.getTestCases())) {
+      measureDraft.setTestCases(convertDateTimeToUTC(measure.getTestCases()));
```

Comment:
I think this will overwrite the results of 
measureDraft.setTestCases(cloneTestCases(measure, measureDraft.getGroups(), accessToken));

Reply:
okay, that means we can avoid double iteration. we can do this in cloneTestCases. we anyways iterate over each test case there. slight optimization

Reply:
@RohitKandimalla @adongare Thanks so much for suggestion, will do.

Reply:
@adongare @RohitKandimalla I moved the logic in cloneTestCases.